command/commands/team.py

```diff
@@ -163,3 +172,35 @@ def handle(self, command, user_id):
 
         else:
             return self.get_help(), 200
+
+    def create_helper(self, param_list):
+        """
+        Create Team and calls GitHub API to create in GitHub.
+
+        If ``param_list[name]`` is not ``None``, will
+        add a display name. If ``param_list[channel] is not
+        ``None``, will add all members of channel in which the
+        command was called into the team.
+        :param param_list: List of parameters for creating team
+        :return: return error message if team created unsuccessfully
+                    otherwise returns success message
+        """
+        try:
+            msg = "new team: {}, ".format(param_list["team_name"])
+            team_id = self.gh.org_create_team()
+            team = Team(team_id, param_list["team_name"], "")
+            if param_list["name"] is not None:
+                msg += "name: {}, ".format(param_list["name"])
+                team.display_name = param_list["name"]
+            if param_list["platform"] is not None:
+                msg += "platform: {}, ".format(param_list["platform"])
+                team.platform = param_list["platform"]
+            if param_list["channel"] is not None:
+                msg += "add channel"
+                # stub: cannot finish until teamMember PR is pushed
+            self.facade.store_team(team)
+            return msg, 200
+        except GithubAPIException as e:
+            logging.error("team created unsuccessfully")
```

Comment:
Maybe `"team creation unsuccessful"`? " "Team created unsuccessfully" sounds strange to my ears. Same with the later instances.